bmi_calculator/lib/widgets/re_calculate_button.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import '../screens/calculator_screen.dart';
+
+class ReCalculateButton extends StatelessWidget {
+  const ReCalculateButton({
+    Key key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: Align(
+        alignment: FractionalOffset.bottomCenter,
+        child: MaterialButton(
+          height: 80,
+          minWidth: 350,
+          color: Theme.of(context).accentColor,
+          onPressed: () {
+            Navigator.push(
+              context,
+              MaterialPageRoute(builder: (context) => CalculatorScreen()),
+            );
+          },
```

Comment:
I would suggest you either poping the page (`Navigator.pop(context)`), or one of the `Navigator` functions that pop the previous page and then push the new one.

I believe by only pushing pages you'll end up with a big stack of pages!